src/components/UserDetailSections/PatronBlock/PatronBlock.js

```diff
@@ -22,6 +22,12 @@ import { calculateSortParams } from '../../util';
 
 import css from './PatronBlock.css';
 
+const PATRON_BLOCKS_COLUMNS = {
+  type: 'type',
+  displayDescription: 'displayDescription',
+  blockedActions: 'blockedActions',
+};
```

Comment:
Probably this constant should be taken out to `src/constants.js` file

Reply:
This constants not reusable i suggest keep them in current file